integration-tests/src/test/java/com/datastax/dse/driver/api/core/graph/GraphPagingIT.java

```diff
@@ -57,7 +57,8 @@
 @RunWith(DataProviderRunner.class)
 public class GraphPagingIT {
 
-  private static final CustomCcmRule CCM_RULE = GraphTestSupport.GRAPH_CCM_RULE_BUILDER.build();
+  // Can't use the shared CCM rule because we pause/resume a node
+  private static final CustomCcmRule CCM_RULE = GraphTestSupport.CCM_BUILDER_WITH_GRAPH.build();
```

Comment:
Of all the graph tests impacted by this changeset, this is the only one that didn't migrate to the shared rule.